hack/update-codegen.sh

```diff
@@ -18,25 +18,33 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-SCRIPT_PACKAGE=k8s.io/cluster-registry
-SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
+SCRIPT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 SCRIPT_BASE=$(cd ${SCRIPT_ROOT}/../..; pwd)
+SCRIPT_PACKAGE=$(basename $(dirname "${SCRIPT_ROOT}"))
 
+# Called on EXIT after the temporary directory is created.
+function clean_up() {
+  if [[ "${TMP_GOPATH}" == "/tmp/gopath"* ]]; then
+    rm -rf "${TMP_GOPATH}"
+  fi
+}
+
```

Comment:
It's more readable if you move line 83 `trap clean_up EXIT` here.

Reply:
Done.